### DISCUSSION TITLE
Looking for a pure Gnome experience with snapshot rollback

### DISCUSSION THREAD

**Discussion by csp5me** [SpiralLinux/SpiralLinux-project#456] [General]
I am a fan of pure Debian and I like the snapshot rollback facility of OpenSUSE. 

So I was curious to look into Spiral Linux. I like the concept of not being dependent on the developer and rely on the Debian repos.

But my personal taste is different than the developer.
I prefer the pure Gnome Wayland experience, instead of an Ubuntu like GUI.

I managed to get rid of the customizations and start Wayland, but could not upgrade properly to Gnome 46 using debian testing.

Would it be possible to add some instructions on the README page to realize this?

**Comment by geckolinux** [comment]
Hi there, what went wrong with the upgrade to Debian Testing?
